demo/src/app/default/default.component.ts

```diff
@@ -1,10 +1,17 @@
 import {Component} from '@angular/core';
 import {environment} from '../../environments/environment';
+import { Snippet } from '../shared/code/snippet';
 
 @Component({
   selector: 'ngbd-default',
   templateUrl: './default.component.html'
 })
 export class DefaultComponent {
   public version: string = environment.version;
+
+  installation = Snippet({
+    lang: 'shell',
+    code: `# recommanded installation: Angular CLI
```

Comment:
typo → recomm**a**nded

But more importantly, I'd:
- remove this comment about CLI altogether, leave just the snippet
- remove the code highlighting, so it would be all black text and `add` word would not be red
- change `See more installation details` to something shorter? `Get started` like bootstrap does?

Something like this, WDYT?

<img width="564" alt="Screen Shot 2020-04-27 at 15 29 05" src="https://user-images.githubusercontent.com/8074436/80377649-db5f8480-889b-11ea-99dd-10ba5d8c9779.png">

Reply:
Or even remove 'Install with Angular CLI' completely, those who know what it is don't need explanation, those who don't → will click on Get Started ?

Reply:
LGTM otherwise, might be nice to have it straight away